src/tcgwars/logic/impl/gen3/DeltaSpecies.groovy

```diff
@@ -3154,7 +3185,7 @@ public enum DeltaSpecies implements LogicCardInfo {
           text "Any damage done to Azumarill by attacks from [R] Pokémon and [W] Pokémon is reduced by 30 (after applying Weakness and Resistance)."
           delayedA {
             before APPLY_ATTACK_DAMAGES, {
-              if(ef.attacker.owner != self.owner && (ef.attacker.types.contains(R) || ef.attacker.types.contains(W))) {
+              if(ef.attacker.types.contains(R) || ef.attacker.types.contains(W)) {
```

Comment:
I'm curious how this would work with confusion damage onto itself

Reply:
Hmm, can't remember when confusion damage was changed to 3 counters. Think it was before R&S, so it would just place the counters and call it a day :P

Reply:
> Confused Pokémon that fail to attack have 3 damage counters placed on them (compared to dealing 20 damage in the Wizards era).

Yup, luckily no weird interactions with that, haha.